kubernetes/kserve/kserve_wrapper/inference_pb2.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
can we generate the code on fly (eg during installation) instead of copy/paste the auto-gen code at here?